azkaban-webserver/src/main/java/azkaban/webapp/servlet/ExecutorServlet.java

```diff
@@ -338,6 +340,24 @@ private void handleExecutionFlowPage(HttpServletRequest req,
       page.render();
       return;
     }
+    
+    Props props = getApplication().getServerProps(); 
+    String execExternalLinkURL = 
+        props.getString("execution.external.link.url", "");
+    String execExternalLinkLabel = 
+        props.getString("execution.external.link.label", "External Analyzer");
+        
+    if (StringUtils.isNotEmpty(execExternalLinkURL)) {
+      StringBuilder builder = new StringBuilder();
+      builder.append(req.getRequestURL());
+      builder.append("?");
+      builder.append(req.getQueryString());
+      String flowExecutionURL = builder.toString();
```

Comment:
Dr. Elephant works with the full URL, other applications need not. I think this will require a change in Dr. Elephant.